AwfulCore/Sources/AwfulCore/Persistence/PostPersistence.swift

```diff
@@ -190,9 +190,10 @@ internal extension ShowPostScrapeResult {
                 in: context,
                 matching: .init("\(\AwfulThread.threadID) = \(id.rawValue)"),
                 configure: { $0.threadID = id.rawValue })
-
-            if !threadTitle.isEmpty, threadTitle != thread.title { thread.title = threadTitle }
-
+            
+            let ignoredPostThreadTitle = "The Something Awful Forums"
+            if !threadTitle.isEmpty, threadTitle != thread.title, threadTitle != ignoredPostThreadTitle { thread.title = threadTitle }
+           
```

Comment:
I'm not sure about this, maybe the thread actually is called "The Something Awful Forums".

This struct is only for scraping that "single post on a page by itself", which we only ever use for ignored posts. I say let's not even bother trying to scrape the thread title and updating it here.

Reply:
Yeah, i had the same thoughts but erred on the side of caution. I also figured that a thread would have to be renamed from something normal to "The Something Awful Forums" and then the ignore post revealing would have to occur... seemed an unlikely series of events

But removal is better if this isn't used for anything else and it should be sorted in my latest commit!